lib/src/query_service.dart

```diff
@@ -87,15 +89,29 @@ class QueryService {
   }
 
   /// Generates query based on user inputs.
-  static RequestArgs _generateDV3Query(QueryType queryType,
-      String nextPageToken, String advertiserId, String insertionOrderId) {
+  static RequestArgs _generateDV3Query(
+      QueryType queryType,
+      String nextPageToken,
+      String advertiserId,
+      String mediaPlanId,
+      String insertionOrderId) {
+    const entityStatusFilter = 'filter=entityStatus="ENTITY_STATUS_ACTIVE"';
+    final mediaPlanFilter = 'filter=campaignId="$mediaPlanId"';
```

Comment:
is this used outside the switch for query by media plan?

Reply:
I guessed not, moving it to the inside then.